### PR TITLE
Documentation about metrics

### PR DESCRIPTION
**Pull request by cincuranet**
Event counters left as a legacy thing on the bottom.

### CODE REVIEW

File: entity-framework/core/logging-events-diagnostics/metrics.md [hunk 1]
```diff
@@ -1,37 +1,106 @@
...
 
 Entity Framework Core (EF Core) exposes continuous numeric metrics which can provide a good indication of your program's health. These metrics can be used for the following purposes:
 
 * Track general database load in realtime as the application is running
 * Expose problematic coding practices which can lead to degraded performance
 * Track down and isolate anomalous program behavior
 
+## Metrics
+
+EF Core reports metrics via the standard <xref:System.Diagnostics.Metrics?displayProperty=nameWithType> API. `Microsoft.EntityFrameworkCore` is the name of the meter.
```
Comment (roji) [inline entity-framework/core/logging-events-diagnostics/metrics.md:21]: Add the following below:

> [!NOTE]
> This feature is being introduced in EF Core 9.0 (in preview). [See event counters below](##event-counters-legacy) for older versions of EF Core.

File: entity-framework/core/logging-events-diagnostics/metrics.md [hunk 2]
```diff
@@ -1,37 +1,106 @@
...
+
+| Name     | Instrument Type | Unit (UCUM) | Description    |
+| -------- | --------------- | ----------- | -------------- |
+| `microsoft.entityframeworkcore.optimistic_concurrency_failures` | ObservableCounter | `{failure}` | Cumulative number of optimistic concurrency failures. |
+
+Available starting in: Entity Framework Core 9.0.
+
+### Additional resources
+
+* [.NET documentation on metrics](/dotnet/core/diagnostics/metrics)
```
Comment (roji) [inline entity-framework/core/logging-events-diagnostics/metrics.md:82]: Maybe point to this in the opening paragraph above rather than down below..

**Review by roji — APPROVED** [review]

**Review by ajcvickers — APPROVED** [review]

**Comment by ajcvickers** [comment]
@cincuranet Two things:

- Should we include/link to this in What's New? Is it available in preview 5?
- You are setting a very dangerous precedent for the team by getting docs done this early!

**Comment by noahfalk** [comment]
I'm proud to be the troublemaker :)

**Comment by roji** [comment]
> You are setting a very dangerous precedent for the team by getting docs done this early!

I agree, this is super bad

**Comment by cincuranet** [comment]
@dotnet-policy-service rerun

**Pull request merged**